.github/workflows/blackduck-scan.yaml

```diff
@@ -0,0 +1,104 @@
+# This central workflow is used to run a Black Duck SCA scan on a repository.
+# It is triggered by the Trigger Blackduck Scan workflow,
+# which is defined in the other repositories' .github/workflows
+name: Execute Blackduck SCA Scan
+
+on:
+  workflow_call:
+    inputs:
+      detector_search_depth:
+        description: 'Optional detector search depth for the Black Duck scan.'
+        required: false
+        type: string
```

Comment:
Can the passed parameter something else then a number/integer? If no, we could use the type `number` to use some kind of GHA input validation